db/migrate/20240503090900_update_rejected_offender_sar_data_to_warehouse.rb

```diff
@@ -0,0 +1,28 @@
+class UpdateRejectedOffenderSARDataToWarehouse < ActiveRecord::Migration[7.1]
```

Comment:
This should be a data migration, not a schema migration.
Use `rails generate data_migration migration_name` to create as per https://github.com/hernanvicente/rails-data-migrations